io/io/src/TFile.cxx

```diff
@@ -3995,6 +4004,7 @@ TFile *TFile::OpenFromCache(const char *name, Option_t *, const char *ftitle,
 /// The file will be downloaded to the directory specified by SetCacheFileDir().
 ///
 /// *The caller is responsible for deleting the pointer.*
+/// A nullptr is returned if the file does not exist.
```

Comment:
```suggestion
/// A nullptr is returned if the file cannot be opened.
```
"if the file does not exist" sounds like `TFile::Open("foo.root", "RECREATE")` returns `nullptr` which is incorrect.

Reply:
Why incorrect? This is what I see:

```
root [0] TFile* abc = TFile::Open("/tmp/blabla.root")
Error in <TFile::TFile>: file /tmp/blabla.root does not exist
(TFile *) nullptr

```

Reply:
I specified now the different behavior when reading or writing.